openstackinabox/models/keystone/model.py

```diff
@@ -314,11 +345,15 @@ def init_database(self):
         # Create an admin user and add the admin token to that user
         self.__admin_tenant_id = self.add_tenant('system',
                                                  'system administrator')
+        self.add_admin_tenant_details()
+
         self.__admin_user_id = self.add_user(self.__admin_tenant_id,
                                              'system',
                                              'system@stackinabox',
                                              'stackinabox',
                                              '537461636b496e41426f78')
+        self.add_admin_user_details()
+
```

Comment:
Same as https://github.com/BenjamenMeyer/openstackinabox/pull/28/files#r31276988 but with respect to the user information...